website/content/en/docs/getting_started.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Getting Started"
+linkTitle: "Getting Started"
+weight: 2
+---
+{{% pageinfo %}}
+* Using Sandbox
+  * [Prerequisites](#prerequisites)
+  * [Set Up](#set-up)
+  * [Next Steps](#next-steps)
+  * [Clean Up](#clean-up)
+* [Service Overview](/docs/service_overview)
```

Comment:
This link seems to be broken

Reply:
Service Overview seems to work for me. Can you confirm it's still broken for you?